lib/java/logging-service-opensearch/src/main/java/module-info.java

```diff
@@ -1,18 +1,14 @@
 module org.enso.logging.service.opensearch {
   requires java.net.http;
-  requires scala.library;
   requires ch.qos.logback.classic;
   requires ch.qos.logback.core;
-  requires static org.enso.logging.service.logback;
+  requires org.enso.logging.service.logback;
   requires org.enso.logging.service;
-  requires org.enso.logging.service.remote;
-  // For jsoniter_scala
-  requires org.enso.scala.wrapper;
+  requires org.enso.logging.service.common;
   requires org.slf4j;
-  requires static org.openide.util.lookup.RELEASE180;
```

Comment:
- great, removing `rg.openide.util.lookup.RELEASE180` dependency was the promise of this PR